README.md

```diff
@@ -13,7 +13,7 @@ Only [semver](http://semver.org/)-style versions with optional prefix are
 supported at this time (major.minor.patch).
 
 `gitversion` will filter all tags of the format
-`<prefix><major>.<minor>.<patch>`, sort them, and increment the requested
+`<prefix><major>.<minor>.<patch>-[prerelease]`, sort them, and increment the requested
```

Comment:
We should document that prerelease bump is using the last commit sha, otherwise the logic LGTM 👍